kitchen/ph5tostationxml.py

```diff
@@ -440,7 +440,9 @@ def create_obs_channel(self, station_list, deployment, cha_code, loc_code,
     
     def get_response_inv(self, obs_channel):
         das = obs_channel.data_logger.serial_number
-        self.ph5.read_das_t(das, reread=False)
+        start = (obs_channel.start_date - datetime.datetime(1970,1,1)).total_seconds()
+        end = (obs_channel.end_date - datetime.datetime(1970,1,1)).total_seconds()
+        self.ph5.read_das_t(das, start, end,  reread=False)

```

Comment:
Although this is correct, I would avoid using `datetime.datetime(1970,1,1)` and instead use `datetime.fromtimestamp(mktime(gmtime(0)))`.

ie.
```
def get_epoch_seconds(date_to_convert):
    return (date_to_convert - datetime.fromtimestamp(mktime(gmtime(0)))).total_seconds()
```
command line
```
>>> datetime.fromtimestamp(mktime(gmtime(0)))
datetime.datetime(1970, 1, 1, 0, 0)
```